app/controllers/admins/users_controller.rb

```diff
@@ -0,0 +1,89 @@
+# frozen_string_literal: true
+
+module Admins
+  class UsersController < AdminsController
+    before_action :user, :super_admin, only: %i[index show edit update destroy]
+    helper_method :super_admin?, :admin?, :last_super_admin?
+
+    def index
+      @users = User.all
+    end
+
+    def show; end
+
+    def new
+      @user = User.new
+      @path = '/admins/users'
+    end
+
+    def create
+      @user = User.new(user_params)
+      @user.role = ROW_USER
+      if @user.save
+        redirect_to root_path, notice: CREATE_SUCCESS
+      else
+        @path = '/admins/users'
+        render :new, status: :unprocessable_entity
+      end
+    end
+
+    def edit
+      @path = "/admins/users/#{@user.id}"
+    end
+
+    def update
+      last_super_admin_update
+    end
+
+    def destroy
+      @user.destroy
+      redirect_to root_path
+    end
+
+    private
+
+    def user
+      @user ||= User.find_by(id: params[:id])
+    end
+
+    def super_admin
+      return true if super_admin?
```

Comment:
if method returns true or false it should have ? at the end. In your case it returns true and redirects and render. Dont do like this. return nil if user is super admin and otherwise just 1 redirect with notice